pisetup.sh

```diff
@@ -11,11 +12,11 @@ lowmem=0
 pi=0
 pimodel=$(uname -m 2>/dev/null || echo unknown)
 pitype=$(tr -d '\0' < /sys/firmware/devicetree/base/model)
+repo_url="https://github.com/sm64pc/sm64pc"
```

Comment:
```suggestion
repo_url="https://github.com/sm64pc/sm64ex"
```

The repo was renamed a while back, so you should use the updated URL, though the old one does still work.

Reply:
Done. Tell me if I missing anything else 🙃